tuxemon/core/tools.py

```diff
@@ -520,6 +521,7 @@ def load_image(rect=None, flags=None):
 def number_or_variable(game, value):
```

Comment:
I know this was added by the previous commit, but I just wanted to point out that the Event system has a Common module to hold things like this function. Of course, if something outside the Event system needs to use this then tools would be the better location.

Reply:
Good to know. For now I think it makes sense to have this in tools.py, as it's a generic function that could have multiple uses throughout the code. At the same time I don't see those syntaxes being used outside of event actions, so I'll wait for more feedback on this.

Reply:
@xirsoi i do agree.  tools.py is a great place for random bits of shared code, but since this particular function deals with parsing strings from events, i do agree that it could be better placed.  for now, i don't think its worth refactoring, but could be done later.